core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -282,6 +287,29 @@ private void registerHttpHostAllowListHandler(final Router router) {
     router.route().handler(new HostAllowListHandler(baseConfig.getHttpHostAllowList()));
   }
 
+  protected void addGenericSignHandler(
```

Comment:
Why is this in the base runner? I thought this was specific to eth2 signing? We already have eth_sign RPC for eth1.

Reply:
👍🏼 , forgot about that. Will make it eth2 specific.